helm/promtail/values.yaml

```diff
@@ -1,412 +1,413 @@
-# -- Overrides the chart's name
-nameOverride: null
+promtail:
```

Comment:
I don't think we override all of these default values.
We should rely on upstream as much as possible.
So, I'd say, unless we need to provide different defaults than upstream, we keep as much default values as possible.
This means removing all the values that we want to keep from upstream from this file.

Reply:
Yes, it would make sense

Reply:
I removed some stuff but I'm not sure about what's left. There is still quite a lot remaining but I can't help thinking some of it might be useful in the future.

Reply:
All of the values might be useful in the future.
But a link to upstream values is enough. We don't plan on maintaining the upstream values file.

Our provided values don't need to show all values that might be useful, but only:
- override values that may be harmful on giantswarm clusters, to provide sane defaults to our users
- if some values **have to** be customized by our users, we can show them so it's easier for our users.

All of the other values should disappear from our file IMO.

Reply:
Done. I've kept only the overwritten values and the cpu/memory usage since we might want to set it right away.